### ISSUE TITLE
Indicate what is being painted

### ISSUE THREAD

**Issue by hanslovsky** [saalfeldlab/paintera#310]
Currently, users have no clue what they are painting based on the overlays during a paint stroke. In addition to the brush size, we should indicate:
 - [ ] the label id that is being painted (both numerically and by a small colored square) for
    - [ ] brush strokes and
    - [ ] flood filll
       - [ ] flood fill should also display the label id that is under the cursor to ensure that users fill the correct id
 - [ ] the special id paint mode:
    - [ ] erase in canvas
    - [ ] paint background

**Comment by hanslovsky** [comment]
cc @cpatrickc

**Comment by igorpisarev** [comment]
@hanslovsky The colored square in the status bar already shows the active fragment and segment ids in a tooltip, which will be used for painting and filling. Do you have something else in mind?

**Comment by hanslovsky** [comment]
@igorpisarev Yes, I would like to have it right by the brush/flood-fill overlay so users can see what is being painted without refocusing. This should be a colored square and the label id as string for painting, and `square -> square (id -> id)` for the flood-fill, where the first square is the max id under the cursor.

For special paint operations (i.e. erase in canvas, paint background), the special case should also be obvious from the overlay.
